protobuf/facts_gathered.proto

```diff
@@ -11,17 +13,15 @@ message FactError {
 message Fact {
   string check_id = 1;
   string name = 2;
-  oneof value {
-    string text_value = 3;
-
-    float numeric_value = 4;
+  oneof fact_value {
+    google.protobuf.Value value = 3;
 
     FactError error_value = 5;
```

Comment:
Maybe now this `error_value` becomes `4`?
Not sure on who the numbers are updated

Reply:
my bad